LanternMobileTestbed/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,12 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <service
+            android:name="org.lantern.mobilesdk.service.LanternService"
+            android:exported="false"
+            android:process=":LanternService"
+            />
```

Comment:
Suppose I'm running multiple apps that integrate Lantern--do we want to have this service run in a global process, and have every app share the same process? It sounds like that can be achieved by making the first letter of the process name lowercase. Whenever you go to start the service, I think if there's already a running process associated, it'll just use that automatically, but I'm not sure. This brings up a good question too: what's the behavior like if I'm running the Lantern VPN app and an app that integrates Lantern as a service simultaneously?